### PR TITLE
Adding Check-In and Check-Out message individually

### LINKED ISSUE #2471: Implementation of Exact Message with Specific Pop-up , Whenever a attendee checks-in or checks-out .
**Is your feature request related to a problem? Please describe.**

- No , not related to a problem , but an enhancement if mentors would like to consider.
- Whenever an attendee checkouts , There should specific pop-up like "Attendee Checkout Successfully" which should be in red dialogue box.
- Whenever an attendee is checked in Dialogue box should say "Attendee checked-in successfully" 

**Describe the solution you'd like**

Implementing this enhancement .

**Additional context**

image
Something Like This

### PR DESCRIPTION
**Pull request by kushthedude**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
It adds individual 'Check-In' and 'Check-Out' message whenever attendee changes his status .
#### Changes proposed in this pull request:

- Added a new action 'ToggleCheckOut'
- Changed button name to 'Check-Out' from 'Cancel'

Fixes #2471

### CODE REVIEW

**Comment by kushthedude** [comment]
@CosmicCoder96 Review Please

**Comment by kushthedude** [comment]
> * ‘Toggle’ means variable is switched between two possible values - an attendee can either be checkedin or checked out at a given point of time. There is no third state. If you make two functions called toggle check in and toggle check out - it makes no sense and code is basically duplicated. Do this inside the same original function.

Did changes @CosmicCoder96

File: app/controllers/events/view/tickets/attendees/list.js [hunk 1]
```diff
@@ -37,19 +37,29 @@ export default Controller.extend({
...
+        attendee.save()
+          .then(() => {
+            this.notify.success(this.get('l10n').t('Attendee Checked-Out Successfully'));
+            this.send('refreshRoute');
+          })
+          .catch(() => {
+            this.notify.error(this.get('l10n').t('An unexpected error has occurred'));
+          });
+      } else {
+        attendee.toggleProperty('isCheckedIn');
```
Comment (CosmicCoder96) [inline app/controllers/events/view/tickets/attendees/list.js]: what's the point of putting this statement in both if and else blocks. Write it once outside both if it is executed irrespective of the condition being true or false.

Comment (kushthedude) [reply app/controllers/events/view/tickets/attendees/list.js]: If the toggle property i would outside the if , else block and then i need to put attendee.save outside too as the functionality will create error and if i put .save() outside if, else  , Build error would occur due to .then and .catch .
That is why i repeated the .toggleProperty in both if , else .
@CosmicCoder96

Comment (CosmicCoder96) [reply app/controllers/events/view/tickets/attendees/list.js]: > If the toggle property I would outside the if 
I don't know what that means.
>  then i need to put attendee.save outside too
Why would you need to put it "outside too"

Comment (kushthedude) [reply app/controllers/events/view/tickets/attendees/list.js]: @CosmicCoder96 Did the changes

**Comment by kushthedude** [comment]
@CosmicCoder96 review

**Comment by kushthedude** [comment]
> ```js
> ```js
> attendee.toggleProperty('isCheckedIn');
> if(attendee.isCheckedIn) {
>   attendee.set('checkinTimes', newCheckinTimes);
> }
> attendee.save()
>   .then(savedAttendee => {
>     this.notify.success(this.get('l10n').t(`Attendee ${savedAttendee.isCheckedIn ? 'Checked-In' : 'Checked-Out'} Successfully`));
>     this.send('refreshRoute');
>   })
>   .catch(() => {
>     this.notify.error(this.get('l10n').t('An unexpected error has occurred'));
>   });
> ```
> 
> 
> This is what I meant.
> ```

I will implement the ternary operator ASAP

**Comment by kushthedude** [comment]
@CosmicCoder96 Implemented the ternary operator. Review Please

**Pull request merged**